.github/workflows/very_good_wear_app.yaml

```diff
@@ -43,25 +43,25 @@ jobs:
         run: |
           dart pub global activate mason_cli
           mason get
-          mason make very_good_wear_app -c very_good_wear_app/config.json -o output --on-conflict overwrite
+          mason make very_good_wear_app -c very_good_wear_app/config.json --on-conflict overwrite
 
       - name: 📦 Install Dependencies
         run: |
           dart pub global activate very_good_cli
-          very_good packages get --recursive output/test_app
+          very_good packages get --recursive very_good_wear_app_output
 
       - name: ✨ Check Formatting
-        run: dart format --set-exit-if-changed output/test_app
+        run: dart format --set-exit-if-changed very_good_wear_app_output
 
       - name: 🕵️ Analyze
-        run: dart analyze --fatal-infos --fatal-warnings output/test_app
+        run: dart analyze --fatal-infos --fatal-warnings very_good_wear_app_output
 
       - name: 🧪 Run Tests
         run: |
-          cd output/test_app
+          cd very_good_wear_app_output
           very_good test -j 4 --recursive --optimization --coverage --test-randomize-ordering-seed random
 
       - name: 📊 Check Code Coverage
         uses: VeryGoodOpenSource/very_good_coverage@v3
         with:
-          path: output/test_app/coverage/lcov.info
+          path: very_good_wear_app_output/coverage/lcov.info
```

Comment:
What's the reason behind the rename of the output path in most of the templates?

Reply:
I noticed that by creating `output/foo` we avoid running mason hooks on the bricks, so I discussed with @wolfenrain that we should instead use a single output directory named `{brick}_output` so mason can run the hooks for it. currently ci didn't validate this so I had to update the workflows as well

Reply:
Interesting thanks for the insights!

Do we know why mason was not running the hooks there?

Reply:
basically because we're nesting the brick output inside another folder so it doesn't resolve the correct directory, instead of running it inside the test_app directory, it runs it on the output (which doesn't have any relevant files at that level, just the test_app folder)